### ISSUE TITLE
I managed to make 77.24% by using step LR scheduler.

### ISSUE THREAD

**Issue by zhaowt61** [jfzhang95/pytorch-deeplab-xception#145]
By using the following settings, I managed to achieve 77.24% at 24 epochs.

Setting:
1 dataset: Pascal + SDB
2 4 opus, BatchSize = 16
3 Backbone: ResNet
4 —freeze-bn False 
5 LR: (replace the function def __call__(self, optimizer, i, epoch, best_pred) in utils/lr_scheduler.py)
   if epoch<10:
          lr = 0.007
   if epoch >= 10 and epoch <=20:
          lr = 0.0007
   if epoch >= 21 and epoch <=40:
          lr = 0.00007

**Comment by clownrat6** [comment]
Thank you, I will get into trouble if there isn't your hint.

**Comment by clownrat6** [comment]
By the way, does the batch size really matter? I train my model only ≈60 mIoU on the voc validation dataset with batch size 4.

**Comment by zhaowt61** [comment]
> By the way, does the batch size really matter? I train my model only ≈60 mIoU on the voc validation dataset with batch size 4.

It is possible. If your maximum batch size is 4, you could divide the learning rate by 4.

**Comment by clownrat6** [comment]
I have tried this training settings, but it's not useful. My training dataset doesn't contain SDB. I can't get better mIoU.

**Comment by zhaowt61** [comment]
> I have tried this training settings, but it's not useful. My training dataset doesn't contain SDB. I can't get better mIoU.

You have to contain SDB to achieve 77%+

**Comment by clownrat6** [comment]
Ok, I get it. Thank you.

**Comment by ImMrMa** [comment]
I follow your choice, but I only get miou at 75%.
I'm using the latest repo and torch with 1.8.1.
Could this be my envs problem or code problem?
